docs/source/exposing.md

```diff
@@ -0,0 +1,267 @@
+
+These IP addresses are only routable within the same Kubernetes cluster, so it's a good fit, if you don't want to expose it to other networks.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+     type: ClusterIP
+```
+
+#### `LoadBalancer` Type
+
+For the 'LoadBalancer' type, the Scylla Operator generates a LoadBalancer Service that directs traffic to a specific node within the ScyllaCluster. 
+On platforms with support for external load balancers, this Service provisions one. 
+The accessibility of this load balancer's address depends on the platform and any customizations made; in some cases it may be reachable from the internal network or public Internet.
+
+LoadBalancer Services are a superset of ClusterIP Services, implying that each LoadBalancer Service also possesses an allocated ClusterIP. 
+These LoadBalancer Services can be configured using the following fields, and these configurations are mirrored across every node Service:
+* externalTrafficPolicy
+* internalTrafficPolicy
+* loadBalancerClass
+* allocateLoadBalancerNodePorts
+
+Check [Kubernetes Service documentation](https://kubernetes.io/docs/concepts/services-networking/service), to know more about these options.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+     type: LoadBalancer
+     loadBalancerClass: my-custom-load-balancer-class
+```
+
+---
+
+### Broadcast Options
+
+Broadcast options controls what is the source of the address being broadcasted to other clients and nodes.
+It's configured independently for clients and nodes because you may want to expose these two types of traffic on different networks, due to costs or security concerns.
+
+#### `PodIP` Type
+
+Address broadcasted to clients/nodes is taken from Pod.
+Because Pod address may be ambiguous, you may provide what's the source by setting `podIP.source`.
+By default, address is taken from Pod `status.PodIP`.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    broadcastOptions:
+       clients:
+         type: PodIP
+         podIP:
+           source: Status
+```
+
+#### `ServiceClusterIP` Type
+
+Address broadcasted to clients/nodes is taken from node dedicated Service, from `spec.ClusterIP` field.
+
+In order to configure it, your `nodeService` template must deploy Service having ClusterIP.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    broadcastOptions:
+       clients:
+         type: ServiceClusterIP
+```
+
+#### `ServiceLoadBalancerIngress` Type
+
+Address broadcasted to clients/nodes is taken from the node dedicated Service, from `status.ingress[0].ipAddress` or `status.ingress[0].hostname` field.
+
+In order to configure it, your `nodeService` template must deploy LoadBalancer Service.
```

Comment:
I'd stick to one form and avoid addressing the users or developers directly - in other words avoid you, your, we etc. unless you want to stick with it throughout the entire document. Applies globally.

Reply:
done